### ISSUE TITLE
Add HeroGlyphs

### ISSUE THREAD

**Issue by thame** [stakersunion/docs#2]
Add to incentives

**Comment by farismln** [comment]
currently there are little to no information about Hero Glyphs. sure it is interesting but I consider to wait until official docs available.

**Comment by thame** [comment]
> currently there are little to no information about Hero Glyphs. sure it is interesting but I consider to wait until official docs available.

Good point, it's not very intuitive but I found a Twitter thread: https://twitter.com/dappnode/status/1787475033113219144
